### PR TITLE
Update Text Submission Action Disclaimer Content

### PR DESCRIPTION
**Pull request by mendelB**
### What does this PR do?

This PR updates the `TextSubmissionAction`'s disclaimer text:

image

image
 

### Any background context you want to provide?
Does this seem like a lot for mobile? should we shrink the font size? (cc @lkpttn )
image

### What are the relevant tickets/cards?

Refs [Pivotal ID #164287413](https://www.pivotaltracker.com/story/show/164287413)
https://www.pivotaltracker.com/story/show/164287413/comments/200590816

### CODE REVIEW

File: resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js [hunk 1]
```diff
@@ -148,8 +148,18 @@ class TextSubmissionAction extends React.Component {
                 {this.props.buttonText}
               </Button>
               <p className="footnote padding-horizontal-md padding-bottom-md">
-                Your submission will be reviewed by a DoSomething.org staffer
-                and added to our public gallery.
+                The data you submit in this form will be handled in accordance
+                with the DoSomething website{' '}
+                {/* Workaround for this jsx-a11y bug https://git.io/fN814 */}
+                {/* eslint-disable-next-line jsx-a11y/anchor-has-content */}
```
Comment (weerd) [inline resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js]: So weird!

Comment (weerd) [reply resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js]: Curious... what if you wrap all the text in a template literal? 🤔 Still gross, but maybe more readable if it works?

Comment (mendelB) [reply resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js]: This was officially [fixed](https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/437), I *think* we just need to update `es-lint` in our [`@dosomething/eslint-config`](@dosomething/eslint-config) package

Comment (mendelB) [reply resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js]: > Curious... what if you wrap all the text in a template literal? 🤔 Still gross, but maybe more readable if it works?

Haha that actually works and is *less* gross! (maybe worth keeping this though so we can easilly spot and update once it's updated on our eslint config?)

Comment (weerd) [reply resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js]: Yeah, fine either way. Mind adding a not to update once `@dosomething/eslint-config` is updated. And maybe throw a quick GH issue in that repo and link here?

**Pull request merged**